### ISSUE TITLE
Use single keys for name of persons and places

### ISSUE THREAD

**Issue by airon90** [Tuxemon/Tuxemon#760]
In order to have a consistent translation, we should use single keys for names and places and use them in keys in order not to have translation errors. For example, we have a key "[misa](https://github.com/Tuxemon/Tuxemon/blob/997a3a87ed18e210e2e9869a422272372cc841fa/mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po#L1241)" to translate the name Misa but in "[a_serious_discussion\n](https://github.com/Tuxemon/Tuxemon/blob/997a3a87ed18e210e2e9869a422272372cc841fa/mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po#L1432)" (is it correct that \n at the end of the key name?) the name Misa is hardcoded. I think it should be something like "I am ${{misa}}, Premier of the Xero Institute for blablabla"

**Comment by kerizane** [comment]
(is it correct that \n at the end of the key name?)
See my comment here:
https://github.com/Tuxemon/Tuxemon/issues/761#issuecomment-618251771

**Comment by bitcraft** [comment]
Yeah I agree with this.  We should be pulling NPC names instead of hard coding them.  The "translated_dialog" event action needs to be able to get more context for substitutions.  In this case, it should lookup NPCs when searching.  I would tweak your great suggestion to something like `slug.attribute` or `misa.name`.
